tix_clock/tix_clock.ino

```diff
@@ -1,9 +1,23 @@
+const int MINUTES_BUTTON_PIN = 5;
+const int HOURS_BUTTON_PIN = 6;
+const int NUMBER_OF_INPUT_PINS = 2;
+int INPUT_PINS[] = {MINUTES_BUTTON_PIN,HOURS_BUTTON_PIN};
+void SetInputPins(int[], int[]);
+
 void setup() {
```

Comment:
The bracket convention for function definition will be:

void function()
{  <- the first bracket should be in a new line
  ...
}